src/ray/rpc/authentication/token_auth_client_interceptor.cc

```diff
@@ -27,18 +27,19 @@
 namespace ray {
 namespace rpc {
 
+RayTokenAuthClientInterceptor::RayTokenAuthClientInterceptor()
+    : token_(AuthenticationTokenLoader::instance().GetToken()) {}
```

Comment:
So... the old code would call instance().GetToken everytime...  This caches it at time of construction. The effect of this is that if the token ever changes for any reason post this construction the change won't take effect.  Though I'm a little unclear if that's likely?  Looking at AuthenticationTokenLoader it seems like this might break if ResetCache is ever called?